executor/set_test.go

```diff
@@ -2124,3 +2124,17 @@ func TestSetMppExchangeCompressionModeVariable(t *testing.T) {
 		require.Equal(t, warnings[0].Err.Error(), "mpp exchange compression won't work under current mpp version 0")
 	}
 }
+
+func TestSetEnableCheckConstraint(t *testing.T) {
```

Comment:
This test function is unnecessary since our system variable framework promises this cast.

Reply:
updated